### PR TITLE
change sriov installation and systemd

### PR DESCRIPTION
**Pull request by alknopfler**
- SRIOV installation using HelmChart CR instead of helm install in systemd
- MTU example added to the static IPs

### CODE REVIEW

**Comment by alknopfler** [comment]
-Tested in Lab
```
kube-system       helm-install-sriov-crd-n77vk                              0/1     Completed          0              58m
kube-system       helm-install-sriov-network-operator-qxqmd                 0/1     Completed          0              58m
...
kube-system       sriov-device-plugin-f7d8b                                 1/1     Running            0              31m
kube-system       sriov-network-config-daemon-7thg6                         1/1     Running            1 (32m ago)    44m
kube-system       sriov-network-operator-56698b464-hdb9s                    1/1     Running            1 (32m ago)    44m
kube-system       sriov-network-operator-sriov-nfd-gc-8c789699f-gdjrq       1/1     Running            1 (32m ago)    44m
kube-system       sriov-network-operator-sriov-nfd-master-c97ff9f8d-8l7rt   1/1     Running            1 (32m ago)    44m
kube-system       sriov-network-operator-sriov-nfd-worker-hbkwq             1/1     Running            1 (32m ago)    44m
```

**Comment by hardys** [comment]
Looks good to me but shall we mark this as a draft until the registry.suse.com charts are available?

File: telco-examples/edge-clusters/dhcp/edge-telco-single-node/telco-capi-single-node-sriov-auto.yaml [hunk 1]
```diff
@@ -88,6 +88,34 @@ spec:
...
+              overwrite: true
+              contents:
+                inline: |
+                  apiVersion: helm.cattle.io/v1
+                  kind: HelmChart
+                  metadata:
+                    name: sriov-crd
+                    namespace: kube-system
+                  spec:
+                    chart: oci://registry.suse.com/edge/charts/sriov-crd-chart
```
Comment (Kristian-ZH) [inline telco-examples/edge-clusters/dhcp/edge-telco-single-node/telco-capi-single-node-sriov-auto.yaml:101]:
```suggestion
                    chart: oci://registry.suse.com/edge/sriov-crd-chart
```

File: telco-examples/edge-clusters/dhcp/edge-telco-single-node/telco-capi-single-node-sriov-auto.yaml [hunk 2]
```diff
@@ -88,6 +88,34 @@ spec:
...
+              overwrite: true
+              contents:
+                inline: |
+                  apiVersion: helm.cattle.io/v1
+                  kind: HelmChart
+                  metadata:
+                    name: sriov-network-operator
+                    namespace: kube-system
+                  spec:
+                    chart: oci://registry.suse.com/edge/charts/sriov-network-operator-chart
```
Comment (Kristian-ZH) [inline telco-examples/edge-clusters/dhcp/edge-telco-single-node/telco-capi-single-node-sriov-auto.yaml:115]:
```suggestion
                    chart: oci://registry.suse.com/edge/sriov-network-operator-chart
```

**Review by Kristian-ZH — APPROVED** [review]

File: telco-examples/edge-clusters/dhcp-less/edge-telco-single-node/telco-capi-single-node-sriov-auto.yaml [hunk 3]
```diff
@@ -88,6 +88,34 @@ spec:
...
+              contents:
+                inline: |
+                  apiVersion: helm.cattle.io/v1
+                  kind: HelmChart
+                  metadata:
+                    name: sriov-network-operator
+                    namespace: kube-system
+                  spec:
+                    chart: oci://registry.suse.com/edge/charts/sriov-network-operator-chart
+                    targetNamespace: kube-system
```
Comment (hardys) [inline telco-examples/edge-clusters/dhcp-less/edge-telco-single-node/telco-capi-single-node-sriov-auto.yaml:116]: Does this need to use `kube-system`?  The [chart docs](https://github.com/suse-edge/charts/tree/main/charts/sriov-network-operator/1.2.4%2Bup0.1.0#deploy-sr-iov-network-operator) describe using a `sriov-network-operator` namespace which would probably be preferable?

**Review by hardys — CHANGES_REQUESTED** [review]

**Comment by hardys** [comment]
One question re the target namespace, otherwise lgtm

**Review by hardys — APPROVED** [review]

**Pull request merged**